<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>598</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>585</width>
    <height>603</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>545</width>
         <height>522</height>
        </rect>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>120</y>
          <width>165</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QLabel" name="currentPlayer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Cambria</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Player 1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="dice">
           <property name="font">
            <font>
             <family>Cambria</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="currentPlayerLabel">
           <property name="font">
            <font>
             <family>Cambria</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>current Player:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="diceLabel">
           <property name="font">
            <font>
             <family>Cambria</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Dice: </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Cambria</family>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Kniffel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>271</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Cambria</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>created by Thomas Fett &amp; Christoph NÃ¼tzel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>95</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Cambria</family>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>change names</string>
        </property>
       </widget>
       <widget class="QTableView" name="leftTableView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>195</width>
          <height>412</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>26</number>
        </attribute>
       </widget>
       <widget class="QTableView" name="rightTableView">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>350</y>
          <width>197</width>
          <height>173</height>
         </rect>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>180</y>
          <width>281</width>
          <height>101</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Kubes</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>19</y>
           <width>261</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="kubesGridLayout"/>
        </widget>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>290</y>
          <width>71</width>
          <height>82</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Cambria</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Roll dices</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="instructionButton">
           <property name="font">
            <font>
             <family>Cambria</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Instruction</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
